modules/animationqt/src/animationeditordockwidgetqt.cpp

```diff
@@ -268,11 +299,69 @@ AnimationEditorDockWidgetQt::AnimationEditorDockWidgetQt(
     }
 
     toolBar->addSeparator();
+
     controller_.AnimationControllerObservable::addObserver(this);
 }
 
 AnimationEditorDockWidgetQt::~AnimationEditorDockWidgetQt() = default;
 
+void AnimationEditorDockWidgetQt::openAnimation() {
+    InviwoFileDialog openFileDialog(this, "Import Animation ...", "animation");
+    openFileDialog.addSidebarPath(PathType::Workspaces);
+    openFileDialog.addExtension("inv", "Inviwo File");
+    openFileDialog.setFileMode(FileMode::AnyFile);
+
+    if (openFileDialog.exec()) {
+        QString path = openFileDialog.selectedFiles().at(0);
+        std::string fileName{utilqt::fromQString(path)};
+        fileName = filesystem::cleanupPath(fileName);
+        if (!filesystem::fileExists(fileName)) {
+            LogError("Could not find file: " << fileName);
+            return;
+        }
+        try {
+            std::ifstream anim{std::string(fileName)};
+            auto app_ = InviwoApplication::getPtr();
+            auto deserializer = app_->getWorkspaceManager()->createWorkspaceDeserializer(
+                anim, std::string(fileName));
+            controller_.pause();
+            deserializer.deserialize("Animation", controller_.getAnimation());
```

Comment:
For this to work the destination network must have the same properties as the source network, which is a tricky precondition I think. I don't think open and save carry the right semantics for this kind of operation, something like import animation. Then it also makes sense to use the same extension. And it is understandable it there are issues...

Reply:
It was actually what I first used, but then changed to match the shortcuts. I have switched back to import/export. 

Trying to open the exported animation will produce a bit of a nasty error ("application might be in a corrupt state"), but I think we can live with it for now as this will not be very common.

Reply:
I really don't want to introduce those kind of bugs willingly into the codebase. That should be fixed.

Reply:
It is more a bug in the workspace deseralization, which is kind of independent of this.

Reply:
No in the regular deserialization we know that all properties exist. Since we gave invariants that are guaranteed by the animation editor and the network. In this use case though you break does invariants.